pkg/apis/componentconfig/types.go

```diff
@@ -712,3 +712,15 @@ type PersistentVolumeRecyclerConfiguration struct {
 	// in a multi-node cluster.
 	IncrementTimeoutHostPath int32 `json:"incrementTimeoutHostPath"`
 }
+
+const (
+	ApiServerConfigMapName string = "apiserver-config"
```

Comment:
Even the configmap name in the system should NOT be constant. This is because when self hosting, doing a rolling update of the config options will require making multiple configmaps.